docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx

```diff
@@ -0,0 +1,172 @@
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-PostgreSQL.png)
+  </TabItem>
+  <TabItem value="SQL Server" label="SQL Server">
+    ![Sandbox Configuration - SQL Server](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-Mssql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-Mssql.png)
+  </TabItem>
+</Tabs>
+
+
+### SingleTenant Configuration
+
+This configuration shows a basic example of a single tenant configuration where
+a single ODS database serves all data going through the API. While this example
+has one ODS for demonstration purpose, single tenant deployments can support
+separate ODS per school years and districts. See [API Client and ODS Instance
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559631/API+Client+and+ODS+Instance+Configuration)
+for more information.
+
+Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-FI ODS Admin Api
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![SingleTenant Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
+  </TabItem>
+  <TabItem value="SQL Server" label="SQL Server">
+    ![SingleTenant Configuration - SQL Server](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+  </TabItem>
+</Tabs>
+
+### SingleTenant with ODS Context Configuration
+
+This configuration shows a basic example of _explicit_ data segmentation
+strategy based on school year, i.e. school year becomes a required part of the
+API route segments. See [Context-Based Routing for Year-Specific
+ODS](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559277/Context-Based+Routing+for+Year-Specific+ODS)
+for more information.
+
+Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-FI ODS Admin Api
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![SingleTenant with ODS Context Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenantContext-PostgreSql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenantContext-PostgreSql.png)
+  </TabItem>
+</Tabs>
+
+### MultiTenant Configuration
+
+This configuration shows a basic example of multiple tenant configuration, with
+one Admin, Security, ODS databases per tenant. See [Multi-Tenant
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559218/Single+and+Multi-Tenant+Configuration)
```

Comment:
Update this link to reference/ods-api/platform-dev-guide/configuration/single-and-multi-tenant-configuration